code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityExtension.java

```diff
@@ -269,11 +268,7 @@ void handleUpdateIdentities(@NonNull final Event event) {
 		final IdentityMap map = IdentityMap.fromXDMMap(eventData);
 
 		if (map == null) {
-			MobileCore.log(
-				LoggingMode.DEBUG,
-				LOG_TAG,
-				"IdentityExtension - Failed to update identifiers as no identifiers were found in the event data."
-			);
+			Log.debug(LOG_TAG, LOG_TAG, "Failed to update identifiers as no identifiers were found in the event data.");
```

Comment:
Should we add the actual class name as LOG_SOURCE instead of com....

Reply:
My bad! Yes, it missed the fact that the LOG_TAG is "EdgeIdentity" and the name of this class is "IdentityExtension". Will change it.